src/qpmodel.jl

```diff
@@ -232,6 +240,8 @@ function NLPModels.objgrad!(qp::AbstractQuadraticModel, x::AbstractVector, g::Ab
   NLPModels.increment!(qp, :neval_grad)
   if typeof(qp.data) <: QPDataCOO
     coo_sym_prod!(qp.data.Hrows, qp.data.Hcols, qp.data.Hvals, x, g)
+  elseif typeof(qp.data.H) <: AbstractLinearOperator
```

Comment:
@dpo  maybe we can delete these lines by adding Symmetric to LinearOperators? `Symmetric(opA)` would return `opA` if the operator is symmetric. Or should we leave this as it is?

Reply:
I'm not sure how `Symmetric(op)` would work since all we have is a function to compute `op * x`. What kind of simplifications were you thinking would be made possible?

Reply:
I was thinking `Symmetric(op) = issymetric(op) ? op : error("not symmetric")`.
This way we could use `mul!(res, Symmetric(op), x)` for LinearOperators and be generic with matrices.

Reply:
Ok. I don't see any problem with that.

Reply:
I opened a PR :
https://github.com/JuliaSmoothOptimizers/QuadraticModels.jl/pull/56